src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
```suggestion
public class Lottery {
```

 We should not extend Lottery class from ColorSupplier because these 2 methods do not have a common state. We can create an instance of ColorSupplier class in Lottery and use it there.